devicetypes/smartthings/smartpower-outlet.src/smartpower-outlet.groovy

```diff
@@ -128,8 +136,9 @@ def refresh() {
 }
 
 def configure() {
-	// Device-Watch allows 2 check-in misses from device
-	sendEvent(name: "checkInterval", value: 60 * 12, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
+	// Device-Watch allows 3 check-in misses from device (plus 1 min lag time)
```

Comment:
I think it will be more clear to make a comment that the checkInterval is being set to a long interval and will be updated to the correct value when the device responds to the configure reporting command. It's just confusing since 3600 seconds doesn't appear to be the max report interval.